solidity/dashboard/src/components/ViewContractInBlockExplorer.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import { useEtherscanUrl } from "../hooks/useEtherscanUrl"
+
+const ViewContractInBlockExplorer = ({ contractAddress, text }) => {
+  const etherscanDefaultUrl = useEtherscanUrl()
+
+  return (
+    <a
+      href={`${etherscanDefaultUrl}/address/${contractAddress}#code`}
```

Comment:
just want to confirm.. we want to navigate to the byte code of the contract? That's why we add #code?

Reply:
exactly, but rather to contract source Code. If the contract is verified, we can check also the source code